src/app/components/index.tsx

```diff
@@ -0,0 +1,30 @@
+import Header from "./Header";
+import HeroSection from "./HeroSection";
+import Logo from "./Logo";
+import Wrapper from "./Wrapper";
+import FeaturedTranscripts from "./FeaturedTranscripts";
+import TranscriptCard, { ExploreTranscriptCard } from "./TranscriptCard";
+import ExploreTranscripts from "./ExploreTranscripts";
+import FooterTop from "./FooterTop";
+import WhyTranscripts from "./WhyTranscripts";
+import FooterComponent from "./FooterComponent";
+import SuggestModal from "./SuggestionModal";
+import ExploreTranscriptClient from "./ExploreTranscriptClient";
+import FeaturedTranscriptClient from "./FeaturedTranscriptClient";
+
+export {
+  Header,
+  HeroSection,
+  Logo,
+  Wrapper,
+  FeaturedTranscripts,
+  FeaturedTranscriptClient,
+  TranscriptCard,
+  ExploreTranscripts,
+  FooterTop,
+  WhyTranscripts,
+  FooterComponent,
+  SuggestModal,
+  ExploreTranscriptClient,
+  ExploreTranscriptCard,
+};
```

Comment:
I like using `index.ts` files as they allow us to export everything from a central location. However, we've been following a different export pattern across the rest of the projects, and I believe it would be best to maintain consistency.

For reference, here's the pattern used in another file ([src](https://github.com/bitcointranscripts/transcription-review-backend/blob/7ae7f37c17fcbab4116ef1b06a377f52c07a322e/api/app/routes/index.ts#L4)): 
```
export * from "./review.routes";
export * from "./transaction.routes";
export * from "./transcript.routes";
export * from "./user.routes";
```

And the alternative we use elsewhere ([src](https://github.com/bitcointranscripts/transcription-review-front-end/blob/25e257b8a0bdf812f5a090ad3d47e15f58cf04aa/src/components/modals/index.ts#L3)):
```
export { default as ReviewGuidelinesModal } from "./ReviewGuidelinesModal";
export { default as SubmitTranscriptModal } from "./SubmitTranscriptModal";
export { default as RestoreOriginalModal } from "./RestoreOriginalModal";
export { default as SuggestModal } from "./SuggestModal";
```

Personally, I prefer the second approach, but I’m open to discussing this further to see what works best for everyone.
cc @Emmanuel-Develops